src/components/PatientList/PatientList.js

```diff
@@ -1,15 +1,53 @@
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import equal from 'deep-equal';
+
 
 import PatientListSelectors from './PatientListSelectors/PatientListSelectors';
+import PatientListResults from './PatientListResults/PatientListResults';
+import { fetchPatients } from '../../actions/patient';
 
-const PatientList = () => {
-  return (
-    <div className="patient-list row">
-      <PatientListSelectors />
-    </div>
-  );
-};
+class PatientList extends Component {
+
+  componentWillReceiveProps(nextProps) {
+    let groupIds = nextProps.population.selectedPopulations.map((p) => p.id);
+    if(nextProps.population.populationSelectorType === 'union' && groupIds.length > 0){
+      groupIds = groupIds.join(',');
+    }
+    // For some reason, moving this check into shouldComponentUpdate doesn't seem to work properly.
+    if (!equal(nextProps.population, this.props.population)) {
```

Comment:
Change to `if (!equal(nextProps.population.populations`, `this.props.populations)) {`.